Mathlib/Data/Polynomial/Degree/Definitions.lean

```diff
@@ -1132,6 +1132,21 @@ theorem eq_C_of_natDegree_eq_zero (h : natDegree p = 0) : p = C (coeff p 0) :=
 theorem eq_C_coeff_zero_iff_natDegree_eq_zero : p = C (p.coeff 0) ↔ p.natDegree = 0 :=
   ⟨fun h ↦ by rw [h, natDegree_C], eq_C_of_natDegree_eq_zero⟩
 
+theorem eq_C_one_of_Monic_natDegree_zero (hf : p.Monic) (hfd : p.natDegree = 0) : p = C 1 := by
```

Comment:
```suggestion
theorem eq_C_one_of_monic_natDegree_zero (hf : p.Monic) (hfd : p.natDegree = 0) : p = C 1 := by
```
perhaps? I'm not 100% sure of the naming convention but it seems strange that you have two different styles next to each other